README.md

```diff
@@ -147,16 +137,15 @@ Once the 3rdParty dependencies are built, you can start building your SDK as fol
 * mkdir cmake-build-x64 (folder names matching pattern cmake-build-* in contrib/cmake are ignored by git)
 * Download and install CMake from https://cmake.org/download/. Mininimum required version is 3.15
 * cd cmake-build-x64
-* Run CMake-gui from this folder
-	* Set your options. Choose the same compiler and confugurations used to generate 3rdParty.
-	* Set your flags, e.g., USE_FREEIMAGE == 0
-	* Generate Projects
-* It should generate all the prjects inside cmake-build-x64 folder.
-* Open the sln file in your Visual Studio and build Debug/Release target as desired.
+* Run CMake-gui from this folder:   cmake-gui ..
+    * Click "Configure"
+	* Choose your compiler etc if it prompts for that (Choose the same compiler and confugurations used to generate 3rdParty)
+	* Set your options and flags (though the defaults should be fine for just SDK), e.g., USE_FREEIMAGE == 0
+	* Make sure UNCHECKED_ITERATORS matches the setting used in your 3rdparty (for debug, off is faster, but that can't be turned off if using Qt)
+	* Click "Configure" again, and then "Generate"
+* It should generate all the projects, and Visual Studio .sln file, inside cmake-build-x64 folder.
```

Comment:
is this indentation on purpose?

Reply:
Adjusted - will double check the rendered version after push